### DISCUSSION TITLE
Provide public API with path to project for plugins

### DISCUSSION THREAD

**Discussion by Czaki** [tox-dev/tox#2948] [Q&A] [answered]
## What's the problem this feature will solve?

After discussion in #2886 I have decided to create a plugin to simplify minimum requirements testing https://pypi.org/project/tox-min-req/. But during development, I meet problems with determining the path to the project. Finally, I decide to use `tox_env.core._root` 
https://github.com/Czaki/tox-min-req/blob/1e32144066ef8f41aae15abb7c94b639357f8be5/tox_min_req/_tox_plugin.py#L45

I also observe that `arguments` contains a string which part is the path to the tox file. But I do not find documentation about this.

## Describe the solution you'd like

Provide public API to determine the path to the project or update documentation of `tox_on_install` or `ToxEnv` with a point to the proper property.

## Alternative Solutions

Maintenance on the side of the plugin.

**Comment by gaborbernat** [comment, marked as answer]
`tox.core['package_root']` is the path ahead per https://tox.wiki/en/latest/config.html#package_root (the tox core config keys are keys to the core object)
